en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+- The Web Application proceeds to make **1 API call to the [Reservations API](#reservationsAPI)**. This is counted as 1 step.
+- The Reservation API then makes **1 API call to the [Bookings API](#bookingsAPI)**. This is counted as **1 step**. 
+- The bookings API makes **1 API call to the Booking Database** to update it with the booking details. This is counted as **1 step**.
+- Upon receiving the booking confirmation, the Reservation API makes **1 call to the [Twilio API](#twillioAPI)** to send an SMS confirmation to the customer. This is counted as **1 step**.
+
+- **Assuming the entire execution time for the reservation process takes 1200ms (1.2s)** , in addition to the above steps,  the total number of extra computation units consumed can be calculated as follows using ***floor((t)/500)) where t is the execution time***: 
+
+          floor(1200/500) = 2 units = 2 steps
+
+Therefore the total number of steps for the reservation process is: 
+
+           4 steps + 2 steps = 6 steps
+
+We can calculate the **total number of steps for searching and reserving a flight** as follows:
+
+- **Search process**: Single execution of a search would cost a total of **2 steps**. 
+- **Reservation process**: Single execution of reservation process would cost a total of **6 steps**. 
+- **Total**: A single execution of the search and reservation process of a flight would cost an organization a total of **8 steps in total**.
+
+!!! note
+    The above cost calculation is dependent on the above architecture and design of the APIs.
+<br/>
+
+### Example 2: REST API Proxy
+An organization exposes a flight tracking API as a managed API on Choreo. This system involves several APIs and databases. The scenario and the APIs in the system are described below:
+
+#### Scenario
+
+To search for real-time flight data, a client will enter search criteria, and search through the web application. Upon receiving the request, the client application (web application) will make a call to the [Flights Status API](#flightStatusAPI), a managed API on Choreo, to search for flight data. Choreo will then invoke the backend API, the [Real-Time Flight API](#realtimeFlightAPI). The backend API will invoke a database to get the information and pass it to the client.
+
+#### Glossary
+
+- <a class="pricing" id="flighStatusAPI">Flight Status API</a>: An external-facing, managed API on Choreo. This API exposes the Real-Time Flight API on Choreo as a managed API.
+
+- <a class="pricing" id="realtimeFlightAPI">Real-TIme Flight API</a>: An unmanaged, external-facing API hosted outside of Choreo.
+
+- <a class="pricing" id="flightsDatabase">Flights Database</a>: A database that stores flight information.
+
+
+#### Get flight status
+
+![Get flight status](../assets/img/references/api-proxy-example1.png)
+
+Let’s see how the steps are calculated for the get flight status process:
+
+- As illustrated above, the client initiates a search request for flight data through the web application. 
```

Comment:
Let's be consistent in the use of capitalization in "web application".

Reply:
kept all instances as "web application".